wakuv2/waku.go

```diff
@@ -342,8 +352,11 @@ func (w *Waku) dnsDiscover(enrtreeAddress string, protocol libp2pproto.ID, apply
 	if !ok {
 		w.dnsAddressCacheLock.Lock()
 		var err error
-		multiaddresses, err = dnsdisc.RetrieveNodes(ctx, enrtreeAddress)
-		w.dnsAddressCache[enrtreeAddress] = multiaddresses
```

Comment:
Was this a bug?

Reply:
I made changes in go-waku sometime ago due to a bug in go-waku that ignored that a ENR could be converted into multiple multiaddresses.